cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -45,14 +45,12 @@ pub async fn create_swap<D, A: ActorState>(
     // construct a generator that watches alpha and beta ledger concurrently
     let mut generator = Gen::new({
         let dependencies = dependencies.clone();
-        |co| {
-            async move {
-                future::try_join(
-                    watch_alpha_ledger(&dependencies, &co, swap.alpha_htlc_params(), at),
-                    watch_beta_ledger(&dependencies, &co, swap.beta_htlc_params(), at),
-                )
-                .await
-            }
+        |co| async move {
```

Comment:
LOL, here it is again (poke @thomaseizinger).  That's three PRs this morning that have included this (and one I noticed of my own the other day).  For reference, @D4nte, this keeps being changed back and forwards from 
```
 |co| { async move {
```
to
```
|co| async move {
```

Reply:
I don't think it keeps being changed back and forth. all PRs I see brings in the new format `|co| async move {` because it was not yet in master.
Now that I rebased this PR you can see that my PR does not bring it back anymore.